modules/javafx.controls/src/main/java/javafx/scene/control/skin/SpinnerSkin.java

```diff
@@ -249,6 +250,12 @@ public void executeAccessibleAction(AccessibleAction action, Object... parameter
                 return null;
             }
         }));
+
+        lh.addChangeListener(control.sceneProperty(), (op) -> {
+            // Stop spinning when sceneProperty is modified
+            behavior.stopSpinning();
+        });
+
```

Comment:
minor: could we get rid of this extra newline?

Reply:
Yes, this can be removed. Since an empty line was there before I had kept it as it is.